decoder/ia_core_coder_init_config.c

```diff
@@ -1392,13 +1400,22 @@ IA_ERRORCODE ia_core_coder_mpegh_3da_config(ia_bit_buf_struct *it_bit_buff,
             return IA_MPEGH_DEC_INIT_FATAL_UNSUPPORTED_CICP_LAYOUT_INDEX;
           }
         }
+        if (ia_signals_3da->group_type[i] == SIG_GROUP_TYPE_CHN)
```

Comment:
Is this check required? This part of code will be entered only when ia_signals_3da->group_type[i] is SIG_GROUP_TYPE_CHN

Reply:
this check was added only for signal group type channel.